Kitodo-API/src/main/java/org/kitodo/config/OPACConfig.java

```diff
@@ -43,8 +44,18 @@ private OPACConfig() {
      * @param parameter String identifying the parameter by name
      * @return value of parameter
      */
-    public static String getConfigValue(String catalogName, String parameter) {
-        return getCatalog(catalogName).getString(parameter);
+    public static String getConfigValue(String catalogName, String parameter) throws ParameterNotFoundException {
+        HierarchicalConfiguration opacConfiguration = getCatalog(catalogName);
+        if (Objects.isNull(opacConfiguration)) {
+            throw new ParameterNotFoundException("No configuration foudn for catalog '" + catalogName + "'!");
```

Comment:
```suggestion
            throw new ParameterNotFoundException("No configuration found for catalog '" + catalogName + "'!");
```

Reply:
This thrown exception is the same exception with what we already have in ConfigException.java.
```suggestion
            throw new ConfigException(catalogName);
```